errors/public.go

```diff
@@ -33,6 +33,8 @@ var (
 	ErrorInvalidURL = errors.New("invalid repo flag") //nolint:errname
 	// ErrorShellParsing indicates there was an error when parsing shell code.
 	ErrorShellParsing = errors.New("error parsing shell code") //nolint:errname
+	// ErrorJobOSParsing indicates there was an error when detecting a job's operating system.
+	ErrorJobOSParsing = errors.New("error parsing job operating system") //nolint:errname
```

Comment:
Don't ignore the linter please, the `errname` exceptions are for the old values

Reply:
Ah, I was just following the pattern! Done!